### ISSUE TITLE
Include guidelines on writing responses

### ISSUE THREAD

**Issue by msamogh** [RasaHQ/rasa-demo#331]
Include some high-level guidelines on the length, tone (e.g., formal vs informal), structure of conversation (e.g., "should nudge the user back into his original goal when they deviate and not do nothing"), etc of the responses and stories.

**Comment by akelad** [comment]
you mean adding this to the rasa docs? I don't think we should, it's up to the end user how they want their bot to talk

**Comment by msamogh** [comment]
No, I was referring to additions/modifications to Sara specifically.
